python/pyspark/ml/wrapper.py

```diff
@@ -34,10 +34,12 @@ class JavaWrapper(Params):
 
     __metaclass__ = ABCMeta
 
-    #: The wrapped Java companion object. Subclasses should initialize
-    #: it properly. The param values in the Java object should be
-    #: synced with the Python wrapper in fit/transform/evaluate/copy.
-    _java_obj = None
```

Comment:
Nevermind, I opened a separate JIRA [SPARK-13937](https://issues.apache.org/jira/browse/SPARK-13937) for this since it is not really related to the issue in this PR.